docs/vs-2015/snippets/cpp/VS_Snippets_Misc/NVC_MFC_StateCollection/cpp/Snippets.txt

```diff
@@ -1,6 +0,0 @@
-File								Snippet #
```

Comment:
@ghogen Is it okay to delete this file? Apparently it isn't used anywhere.

Reply:
OK, it probably should never have been checked in to begin with.